datafusion/sqllogictest/regenerate/src/engines/datafusion_engine/runner.rs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
this is pretty brutal -- I edited these files so they compiled and then copied them to the regenerate location, similarly to how it is tone for the runner.rs